Receiving the error "submission"

shannonfromomaha opened the pull request:
NameError at /assignment/1
undefined local variable or method `submission' for #<Student id: 2, name: "Heather Praisewater"> Did you mean? submissions submissions=

I searched and found that the grade.rb had :submission whereas all other programs used :submissions, plural. i changed them both to :submissions and ran it again and I get an identical error.

My question is: you said not to undo our changes, but if we don't get at least a different error message, can we assume the change we made was not what ruby wanted to work? in that case, would you suggest undoing that work, or going forward to see if you missed another one?

shannonfromomaha committed:
Recieving the error
NameError at /assignment/1
undefined local variable or method `submission' for #<Student id: 2, name: "Heather Praisewater"> Did you mean? submissions submissions=

I searched and found that the grade.rb had :submission whereas all other programs used :submissions, plural. i changed them both to :submissions and ran it again and I get an identical error.

My question is: you said not to undo our changes, but if we don't get at least a different error message, can we assume the change we made was not what ruby wanted to work? in that case, would you suggest undoing that work, or going forward to see if you missed another one?

halfghaninne commented:
I think that this is a good case for undoing the change from "submission" to "submissions" and trying something different.

shannonfromomaha committed:
Got it to load individual assignment pages! WHOOP

shannonfromomaha committed:
everything is there except the grades! (i'm sure this will be a breeze...sure sure)

shannonfromomaha committed:
YEAH YEAH YEAH YEAH YEAH
look at those tables! those tables are beautiful!
i think i have more to do but seriously. those student tables are beautiful.

shannonfromomaha commented:
the assignment doesn't explicitly say we need to fill in the "overall grade" column of the table on the homepage...but are we supposed to?

halfghaninne commented:
Yeah, if you've moved past the errors!

shannonfromomaha commented:
ActiveRecord::ConnectionTimeoutError at /
could not obtain a database connection within 5.000 seconds (waited 5.003 seconds)

this...doesn't seem like something i did?

shannonfromomaha committed:
not actually sure what i did to disconnect just this page from the server

shannonfromomaha committed:
FINISHED!

sumeetjain commented:
:+1:
